sidecars/asciidoc/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020 Red Hat, Inc.
+# Copyright (c) 2020-2021 Red Hat, Inc.
```

Comment:
I assume this is a test change?

Reply:
yes once I have something that works I will remove it 👍

Reply:
:+1:

Reply:
ok it works :-)